docs/content/Getting-Started/Cloud/06-Learn-more.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Learn more
+permalink: /getting-started/cloud/learn-more
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 7
+isDisableFeedbackBlock: true
+---
```

Comment:
I think much more might be added here. Team collaboration in Cube Cloud, query debugging, etc. Basically everything from the Cloud section of docs (I also think it shoudld be merged, we don't really need separate "Cube Cloud docs".)